dspace/config/modules/authentication-password.cfg

```diff
@@ -31,4 +31,4 @@
 # SHA-256, SHA-384, and SHA-512 should be available, but you may have 
 # installed others.  If not set, SHA-512 will be used.
 # digestAlgorithm = SHA-512
-domain.valid = @ubc.ca, @stat.ubc.ca
+domain.valid = @ubc.ca, @stat.ubc.ca, @gmail.com
```

Comment:
please leave those changed to your local. The value here might be used in prod, so we don't want people register using their gmail.